en/integrations/plugin-packs/procedures/applications-ibmmq-mqi.md

```diff
@@ -0,0 +1,162 @@
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **IBM MQ MQI** resources:
+
+```bash
+yum install centreon-plugin-Applications-Ibmmq-Mqi
+```
+
+2. On the Centreon Web interface, install the **IBM MQ MQI** Centreon Plugin Pack on the **Configuration > Plugin Packs** page.
+
+<!--Offline License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **IBM MQ MQI** resources:
+
+```bash
+yum install centreon-plugin-Applications-Ibmmq-Mqi
+```
+
+2. Install the **IBM MQ MQI** Centreon Plugin Pack RPM on the Centreon Central server:
+
+ ```bash
+yum install centreon-pack-applications-ibmmq-mqi
+```
+
+3. On the Centreon Web interface, install the **IBM MQ MQI** Centreon Plugin Pack on the **Configuration > Plugin Packs** page.
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
+
+* Log into Centreon and add a new Host through **Configuration > Hosts**
+* Fill the **Name**, **Alias** & **IP Address / DNS** fields according to your **IBM MQ MQI** server settings
+* Select the **App-Ibmmq-Mqi-custom** template to apply to the Host
+* Once the template is applied, fill in the corresponding macros. Some macros are mandatory.
+
+| Mandatory | Name                 | Description                                                                                     |
+|:----------|:---------------------|:------------------------------------------------------------------------------------------------|
+|           | IBMMQMQIEXTRAOPTIONS | (Default: '--runas=centreon')                                                                   |
+|           | IBMMQMQIPORT         | (Default: '1414')                                                                               |
+|           | EXTRAOPTIONS         | (Default: 'Any extra option you may want to add to every command\_line (eg. a --verbose flag)') |
+
+## How to check in the CLI that the configuration is OK and what are the main options for? 
+
+Once the plugin is installed, log into your Centreon Poller CLI using the 
+**centreon-engine** user account and test the Plugin by running the following 
+command:
+
+```bash
+/usr/lib/centreon/plugins//centreon_ibmmq_mqi.pl \
+    --plugin=apps::mq::ibmmq::mqi::plugin \
+    --mode=queues \
+    --hostname='10.0.0.1' \
+    --channel='' \
+    --port='1414' \
+    --runas=centreon \
+    --filter-name='^(?!(SYSTEM|PERL.COMMAND))' \
+    --warning-connections-input='' \
+    --critical-connections-input='' \
+    --warning-messages-depth='200' \
+    --critical-messages-depth='' \
+    --warning-message-oldest='' \
+    --critical-message-oldest='3600' \
+    --verbose \
+    --use-new-perfdata 
+```
+
+The expected command output is shown below:
+
+```bash
+WARNING: current input connections: 9000 | 'queue.connections.input.count'=9000;200;;0; 'queue.messages.depth.count'=2400;;;0; 'queue.message.oldest.seconds'=9000;;;; 
```

Comment:
The thresholds and the output are not lined up. Same thing for the explanation below